meta-mender-c444/templates/local.conf.append

```diff
@@ -14,8 +14,8 @@ PREFERRED_PROVIDER_u-boot = "u-boot-imx"
 IMAGE_CLASSES += "mender-setup-c444"
 
 # Uncomment the following 2 lines to boot from eMMC on imx8mq-itx-p-c444
```

Comment:
Updating the boot medium to internal eMMC is a good idea 💡 . But can we update the `comment` as well?
```suggestion
# Boot from eMMC
```